pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>17</jdk.version>
+        <jdk.version>11</jdk.version>
```

Comment:
It's not critical for this task but I really recommend to uplift your local java version to 17 as soon your tasks will have problem with compilation